app/src/androidTest/java/com/mapbox/maps/testapp/gestures/GestureInterceptedOnViewAnnotationTest.kt

```diff
@@ -0,0 +1,153 @@
+package com.mapbox.maps.testapp.gestures
+
+import android.view.LayoutInflater
+import android.view.View
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.platform.app.InstrumentationRegistry
+import com.mapbox.geojson.Point
+import com.mapbox.maps.ScreenCoordinate
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.plugin.delegates.listeners.OnMapIdleListener
+import com.mapbox.maps.plugin.gestures.gestures
+import com.mapbox.maps.testapp.BaseMapTest
+import com.mapbox.maps.testapp.test.R
+import com.mapbox.maps.viewannotation.OnViewAnnotationUpdatedListener
+import com.mapbox.maps.viewannotation.viewAnnotationOptions
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+
+/**
+ * Tests gestures intercepted on view annotations.
+ */
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class GestureInterceptedOnViewAnnotationTest : BaseMapTest() {
+
+  private fun withLatch(
+    count: Int = 1,
+    timeout: Long = 3000,
+    lambda: (CountDownLatch) -> Unit
+  ) {
+    val latch = CountDownLatch(count)
+    lambda(latch)
+    if (!latch.await(timeout, TimeUnit.MILLISECONDS)) {
+      throw TimeoutException()
+    }
+  }
+
+  @Test
+  fun clickableViewAnnotationDoesNotTriggerLongTouchAfterwards() {
+    val cameraCenter = Point.fromLngLat(-122.44121, 37.76132)
+
+    var longClicks = 0
+
+    withLatch { latch ->
+      rule.scenario.onActivity {
+        it.runOnUiThread {
+          mapView.gestures.addOnMapLongClickListener {
+            longClicks++
+            return@addOnMapLongClickListener true
+          }
+
+          mapboxMap.setCamera(
+            cameraOptions {
+              center(cameraCenter)
+            }
+          )
+
+          mapView.viewAnnotationManager.apply {
+
+            // xml layout resource is only available within instrumentationContext,
+            // while mapView uses targetContext hence fails to inflate if used via resId
+            val viewAnnotation = LayoutInflater.from(InstrumentationRegistry.getInstrumentation().context).inflate(R.layout.view_annotation, mapView, false)
```

Comment:
Weirdly `targetContext` fails to inflate the view_annotation that is added to the androidText source set, though it's working in the sdk androidTest e.g. https://github.com/mapbox/mapbox-maps-android/blob/main/sdk/src/androidTest/java/com/mapbox/maps/ViewAnnotationTest.kt. Guess activity is setup differently (part of instrumentation app and not the main one https://github.com/mapbox/mapbox-maps-android/blob/main/sdk/src/androidTest/java/com/mapbox/maps/EmptyActivity.kt]), decided to inflate explicitly with instrContext.